rllib/utils/metrics/metrics_logger.py

```diff
@@ -34,16 +34,71 @@ class MetricsLogger:
 
     .. testcode::
 
+        import time
         from ray.rllib.utils.metrics.metrics_logger import MetricsLogger
+        from ray.rllib.utils.test_utils import check
 
         logger = MetricsLogger()
 
-        # Log n simple float values under the "loss" key. By default, all logged values
-        # under that key are averaged over once `reduce()` is called.
-        logger.log_value("loss", 0.001)
-        logger.log_value("loss", 0.002)
-        logger.log_value("loss", 0.003)
+        # 1) Logging float values (mean over window):
+        # Log some loss under the "loss" key. By default, all logged values
+        # under that key are averaged and reported back, once `reduce()` is called.
+        logger.log_value("loss", 0.001, reduce="mean", window=10)
+        logger.log_value("loss", 0.002)  # <- no need to repeat arg/options on same key
         # Peek at the current (reduced) value of "loss":
+        check(logger.peek("loss"), 0.0015)  # <- expect average value
```

Comment:
We should add to the docs sth like an FAQ for MetricsLogger. Like "I see NaN values in my Evaluation results for each non-evaluated Iteration. What shall I do?"